### PR TITLE
Add CNCF project governance

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Adding the required governance for the project to become part of the CNCF.

#### Which issue(s) this PR fixes:

Refers to #240 
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
PTAL @cgwalters @flavio @thomastaylor312 @utam0k @yihuaf @jprendes @Furisto @YJDoc2

**Review by YJDoc2 — APPROVED** [review]
lgtm :+1:
Thanks @saschagrunert !

**Review by cgwalters — APPROVED** [review]

**Review by flavio — APPROVED** [review]
Sorry for being late, LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+We're extremely grateful for security researchers and users that report
+vulnerabilities to the community. All reports are thoroughly investigated by a
+set of community volunteers.
+
+## Report a Vulnerability
+
+<!-- TODO: the mailing list has to be requested -->
+
+To make a report, email the vulnerability to the private
+[cncf-oci-spec-rs-security@lists.cncf.io](mailto:cncf-crio-security@lists.cncf.io) list
```
Comment (jprendes) [inline SECURITY.md:15]: I assume that these two addresses not matching is linked to the above TODO, right?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+| Name            | Employer           | GitHub handle    | Responsibilities        |
+| --------------- | ------------------ | ---------------- | ----------------------- |
+| Colin Walters   | Red Hat            | @cgwalters       | Approver and Maintainer |
+| Flavio Castelli | SUSE               | @flavio          | Approver and Maintainer |
+| Sascha Grunert  | Red Hat            | @saschagrunert   | Approver and Maintainer |
+| Taylor Thomas   | Cosmonic           | @thomastaylor312 | Approver and Maintainer |
+| Toru Komatsu    | Preferred Networks | @utam0k          | Approver and Maintainer |
+| Eric Fang       | Independent        | @yihuaf          | Maintainer              |
+| Jorge Prendes   | Independent        | @jprendes        | Maintainer              |
```
Comment (jprendes) [inline MAINTAINERS.md:11]:
```suggestion
| Jorge Prendes   | Microsoft          | @jprendes        | Maintainer              |
```

**Review by jprendes — CHANGES_REQUESTED** [review]

**Comment by jprendes** [comment]
Ah! Slightly too late, sorry! 😅
I'll create a PR.

**Comment by saschagrunert** [comment]
Thank you @jprendes! :pray:

**Review by utam0k — APPROVED** [review]

**Pull request merged**